packages/financial-templates-lib/src/price-feed/MedianizerPriceFeed.js

```diff
@@ -72,6 +72,16 @@ class MedianizerPriceFeed extends PriceFeedInterface {
     return Math.max(...lastUpdateTimes);
   }
 
+  // Gets the decimals of the medianized price feeds. Errors out if any price feed had a different number of decimals.
+  getPriceFeedDecimals() {
+    const priceFeedDecimals = this.priceFeeds.map(priceFeed => priceFeed.getPriceFeedDecimals());
+    if (priceFeedDecimals.every((val, i, arr) => val !== arr[0])) {
```

Comment:
Should this be `any` rather than `every`?

Also might be slightly simpler/more readable to leave out the latter two args and use `priceFeedDecimals[0]` (or pull that value into a named variable before the call), but I could see arguments for doing it the way you have it too.

Reply:
I think we want `.every` as this method [tests](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) whether all elements in the array pass the test implemented by the provided function. We want to be sure that _all_ decimals within the `MedianizedPrice` feed match their precision else it will break the downstream computation. This syntax is saying that `every` position within the array should equal the 0th element within the array. How would you want to structure it with `.any`?

Regarding your second point, I agree that will help with readability. I've changed it accordingly.

Reply:
Sorry, I should've been clearer -- I think the reason it should be `any` is because you're checking for the error case, not the success case in this _if_. I would agree with everything you said above if you were checking for === in the condition, but you are looking for "any" that do _not_ match your desired pattern if I'm reading the code correctly.

Reply:
To frame it differently, i think in its current form, this if will _always_ return false and therefore never hit the `throw` because you're effectively doing `if (priceFeedDecimals[0] !== priceFeedDecimals[0] && ... )` which will always short-circuit and return false.

Reply:
you are 100% right, sorry for the confusion. I've updated it accordingly and added and added a test to validate the behavior with different feeds.